operations/cancellationpolicies.md

```diff
@@ -102,6 +100,176 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `RelativeFee` | number | required | Relative value of the fee, as a percentage of the reservation price. |
 | `IsActive` | boolean | required | Whether the cancellation policy is still active. |
 
+## New get all cancellation policies
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Given reservation draft parameters or reservation ids, returns applicable Cancellation Policies.
+
+The reservation draft parameters are used when the reservation does not yet exist. This is useful if information about cancellation policies is needed before the reservation is confirmed.
+
+For existing reservations, supply the reservation ids. This is because the cancellation policies are locked per reservation when the reservation is created. That means the reservaton's cancellation policies are not changed when cancellation policy is created, updated or deleted.
+
+If the response do not contain any cancellation policy for any given reservation draft parameters or reservation id, no cancellation policies apply.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/cancellationPolicies/getAll/2024-09-02`
```

Comment:
Not fully aware of use cases and external contracts, but mixing drafts and confirmed reservations with different format of data sounds like overcomplication (complex documentation, complex support). Mb if drafts is needed we need separate endpoint with time limits and rates only. especially if confirmed reservations have fixed policies from some snapshot (as I understand)

Reply:
also getAll usually a request without id parameters in it, this is more like just get with the time interval parameters